crates/sozo/src/ops/events.rs

```diff
@@ -0,0 +1,32 @@
+use starknet::{
+    core::types::{BlockId, EventFilter},
+    core::utils::starknet_keccak,
+    providers::Provider,
+};
+use toml::Value;
+
+use crate::commands::events::EventsArgs;
+use anyhow::Result;
+
+pub async fn execute(args: EventsArgs, env_metadata: Option<Value>) -> Result<()> {
+    let EventsArgs { chunk_size, starknet, world, from_block, to_block, events } = args;
+
+    let from_block = from_block.map(|num| BlockId::Number(num));
+    let to_block = to_block.map(|num| BlockId::Number(num));
+    let keys = events.map(|e| {
+        let mut ret = vec![];
+        for event in e {
+            ret.push(vec![starknet_keccak(event.as_bytes())]);
+        }
+        ret
+    });
+
+    let provider = starknet.provider(env_metadata.as_ref())?;
+    let event_filter = EventFilter { from_block, to_block, address: world.world_address, keys };
+
+    let res = provider.get_events(event_filter, None, chunk_size).await?;
+
+    let value = serde_json::to_value(res)?;
+    println!("{}", serde_json::to_string_pretty(&value)?);
```

Comment:
could we update this to use the logging framework migrate etc use?

Reply:
do you mean adding [`with_context`](https://github.com/dojoengine/dojo/blob/efb3cf6841b679fd94204ace10147a8e84c5fc8a/crates/sozo/src/ops/execute.rs#L17) where a method can return an error?

Reply:
@tarrencev should we update all commands to use scarb's logging framework? currently only `migrate` uses it as it displays bunch of info so adding some filtering to it make sense. but other cmds only output once tho... but i guess it doesn't hurt to use it still??

Reply:
I think it is nice to use to provide a consistent experience